app/workers/transactional_email_analytics_job.rb

```diff
@@ -36,6 +36,19 @@ def perform
     end
   end
 
+  # constantize all the mailers so TransactionalEmailMailer.descendants has something
+  def self.all_mailers
+    [
+      FailedClaimsReportMailer,
+      SpoolSubmissionsReportMailer,
+      TransactionalEmailMailer,
+      ApplicationMailer,
+      DirectDepositMailer,
+      HcaSubmissionFailureMailer,
+      YearToDateReportMailer
```

Comment:
I haven't been following closely the changes to `TransactionalEmail` related classes, so apologies if I'm missing something, but:
"Transactional" emails are supposed to be emails that are generated as a result of a "transaction" with an end user.  They are not reports or other mass mailers to stakeholders; they are emails sent to a single end user (e.g. veteran).
Maybe I didn't choose the clearest name; naming things is hard.
The two mailers that are subclasses of `TransactionalEmailMailer` are `DirectDepositMailer` and `HCASubmissionFailureMailer`.
These emails are generated as results of form submissions.
The bodies of the emails include the end user's unique Google Analytics tracking ID.
To complete the picture for GA, this job runs nightly and reports the delivery status of each email to GA.
This job is irrelevant for any mailer class other than `DirectDepositMailer` and `HcaSubmissionFailureMailer`.

I tried to set this up so it'd be easy to add additional "transactional" mailers in the future:
- Create a new mailer
- make it a subclass of `TransactionalEmailMailer`
- define the same constants you see in `DirectDepositMailer` and `HcaSubmissionFailureMailer`
- create your email view (make sure to include the users' GA tracking ID)
 Done.  Now, you have automatic GA tracking and nightly reporting to GA for you end user destined emails.

Reply:
Just realized there's a new `TransactionalEmailMailer` subclass: `SchoolCertifyingOfficialsMailer`